src/shapes.cpp

```diff
@@ -564,6 +556,70 @@ void Mesh::computeVertexNormals()
   }
 }
 
+void Mesh::computeWeightedVertexNormals()
```

Comment:
As far as I see, this method replaces the old method `computeVertexNormals()`.

Reply:
Replaced current method rather than adding a new method